FAQs.md

```diff
@@ -1,113 +1,135 @@
 
-14. **What is sleep timer?**
 
-The sleep timer is a feature that shuts down a device or an app after a preset time has elapsed. On Jockey music player, this feature is intended to let users listen to their favorite music while they sleep. The player shuts down after time has elapsed.
+14. **What is the "sleep timer"?**
 
+The sleep timer is a feature that shuts down a device or an app after a preset time has elapsed.
+On Jockey, this feature is intended to let users listen to their favorite music while they sleep.
+The player shuts down after the set time has elapsed.
 
-15. **Where is the sleep timer feature found and How can I use the sleep timer feature?**
 
-The sleep timer feature option is only visible on the now playing page when the 3 dots on the top right side is clicked. To use the sleep timer, 
-click on the 3 dots on the top right side
-select sleep timer and move the pointer on the roll to set time. The maximum time is 120 minutes(2 hours)
+15. **Where is the sleep timer feature found and how can I use it?**
 
+The sleep timer feature option is only visible on the "now playing" page when the 3 dots are clicked.
+To use the sleep timer, click on the 3 dots.
+Select sleep timer and move the pointer on the roll to set the time.
+The maximum time is 120 minutes (2 hours).
 
-16. **Does Jockey music player contain ads?**
 
-No, the app doesn't contain any ads or unnecessary pop-ups
+16. **Does Jockey contain ads?**
 
-17. **Do I have to pay for any feature on Jockey music player ?**
+No, the app doesn't contain any ads or unnecessary pop-ups.
+Check the source code to see for yourself.
 
-No, all the features and tools on Jockey music player are available for free.
 
-18. **What device operating system can use Jockey?**
+17. **Do I have to pay for any features?**
 
-Jockey is an android software meaning Jockey is only available to users using an a device running on Android OS. Android 5.0+
+No, all the features and tools are available for free, like they should be.
 
 
+18. **What device and operating system can use Jockey?**
 
-19. **Where can I download Jockey Music Player?**
+Jockey is an Android application, meaning Jockey can only be run on a device running the Android OS.
+Jockey supports Android 5.0 and above. Go to your phone's settings' "About" page to see your OS version.
 
-The latest release of Jockey can be downloaded from [google playstore](https://play.google.com/store/apps/details?id=com.marverenic.music&hl=en). To view the latest releases and all releases of Jockey music player, click [here](https://github.com/marverenic/Jockey/releases)
 
+19. **Where can I download Jockey?**
 
-If there are other questions you think should be added to this FAQs document, feel free to open an issue or add the question(s) to this document and make a pull request.
+The latest release of Jockey can be downloaded from the [Google Play Store](https://play.google.com/store/apps/details?id=com.marverenic.music) and the "releases" page.
```

Comment:
Nit: /s/and/or. Maybe also add a link to the releases tab? Not sure if it's possible to do a generic permalink.

```suggestion
The latest release of Jockey can be downloaded from the [Google Play Store](https://play.google.com/store/apps/details?id=com.marverenic.music) or the "releases" page.
```